sdk/core/core-tracing/src/interfaces.ts

```diff
@@ -79,4 +92,36 @@ export interface OperationTracingOptions {
    * OpenTelemetry SpanOptions used to create a span when tracing is enabled.
    */
   spanOptions?: SpanOptions;
+
+  /**
+   * OpenTelemetry context to use for created Spans.
+   */
+  context?: Context;
+}
+
+/**
+ * OpenTelemetry compatible interface for Context
```

Comment:
if this is compatible, can we use it for all of our public surface?

Reply:
Migrated everything but Tracer. I'll hold that for another PR since this one is going on forever at this point.